### ISSUE TITLE
Disconnected with "ERROR_QUEUE_NOT_FOUND"

### ISSUE THREAD

**Issue by Gizmokid2005** [dequis/purple-facebook#439]
I have recently started seeing very frequent disconnects from my Facebook account with the only messages in the log being:

(10:07:26) connection: Connection error on 10D056C8 (reason: 0 description: ERROR_QUEUE_NOT_FOUND)

A reconnect usually works just fine, but it will disconnect regularly.

Pidgin 2.13.0 (libpurple 2.13.0)
f07927902113041d6fa291293d9205ea4f05e9e6

Facebook
	Author: (null)
	Version: 20180323~ef6ae47~9ff9acf9fa14~145
	Website: https://github.com/dequis/purple-facebook
	ID String: prpl-facebook
	Loadable: Yes
	Loaded: Yes

**Comment by Gizmokid2005** [comment]
This does seem to be limited to only happening when connected from multiple instances of Pidgin.

**Comment by dequis** [comment]
Are those instances of pidgin sharing configs? (same computer, copied config across computers, dropboxed etc)

**Comment by Gizmokid2005** [comment]
Nope. They're on different computers, networks, locations, and have no syncing between them. The config was originally copied from one to the other, but only initially.

**Comment by dequis** [comment]
If it was originally copied after the purple-facebook pidgin account was created, remove the account in one of them and create a new one.

**Comment by Gizmokid2005** [comment]
Is there shared resource or guid of some sort that gets created and needs to be completely unique?

I'll definitely give it a shot and report back.

**Comment by dequis** [comment]
Yeah, a few, as hidden settings in accounts.xml (I don't remember off the top of my head). Things get weird when they are reused.

**Comment by Gizmokid2005** [comment]
I can confirm after recreating the account on my machine that after about 2 hours of use, so far I haven't run into this issue once. Thanks for the help @dequis!

Is this worth maybe dropping in the FAQ/Wiki?

**Comment by Malvineous** [comment]
Thanks for this, I also just came across the same issue with my config that is sync'd between multiple machines.  Removing and re-adding the account did the trick.
